compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmPackageGen.java

```diff
@@ -596,9 +591,7 @@ private void linkGlobalVars(BIRPackage module, String pkgName, String initClass,
         globalVarClassMap.put(pkgName + LOCK_STORE_VAR_NAME, initClass);
     }
 
-    private void linkTypeDefinitions(BIRPackage module, String pkgName,
-                                     InteropValidator interopValidator,
-                                     boolean isEntry, String orgName,
+    private void linkTypeDefinitions(BIRPackage module, String pkgName, boolean isEntry, String orgName,
```

Comment:
Can we check whether pkgName and orgName would be same values that can be used from `BIRPackage module` here?